lib/line/bot/client.rb

```diff
@@ -396,7 +414,7 @@ def link_user_rich_menu(user_id, rich_menu_id)
 
       # Unlink a rich menu from a user
       #
-      # @param user_id [String] ID of the user
+      # @param user_id [String] User Id ID of the user
```

Comment:
ID of the user?

Reply:
wow, I failed to replace the words...
I'll fix it.